### PR TITLE
Fix restart kusama

### PR DESCRIPTION
**Pull request by ironoa**

### CODE REVIEW

File: charts/polkadot-watcher/templates/alertrules.yaml [hunk 1]
```diff
@@ -44,12 +44,20 @@ spec:
       labels:
         severity: warning
         origin: {{ .Values.origin }}
-    - alert: ValidatorOfflineSession
+    - alert: ValidatorOfflineSessionLong
       annotations:
         message: 'Target {{`{{ $labels.name }}`}} has either not authored any block or sent any heartbeat yet in this session'
       expr: polkadot_offline_validator_session_reports_state > 0
```
Comment (ironoa) [inline charts/polkadot-watcher/templates/alertrules.yaml:50]: Before the setup was 3 minutes... I realized that we had a lot of criticals solving themselves after about five minutes. I thought to place a Short warning at 3+5 and a critical at 10

Comment (fgimenez) [reply charts/polkadot-watcher/templates/alertrules.yaml:50]: Ok sounds good

File: src/subscriber.ts [hunk 2]
```diff
@@ -129,29 +126,61 @@ export class Subscriber {
...
+              this.promClient.resetTotalValidatorOfflineReports(account.name);
+          }
+      }
+    }
+
+    private async _sessionOfflineHandler(header: Header): Promise<void> {
+      const isHeartbeatExpected = await this._isHeadAfterHeartbeatBlockThreshold(header)
+      const sessionIndex = await this.api.query.session.currentIndex()
+
+     this.validators.forEach(async account => {
```
Comment (ironoa) [inline src/subscriber.ts:175]: removed await asyncForEach util, not needed here imo

File: test/subscriber.ts [hunk 3]
```diff
@@ -52,7 +52,7 @@ const testRPC = new TestPolkadotRPC();
 
 let subject: Subscriber;
 
-function delay(ms: number) {
+function delay(ms: number): Promise<unknown> {
```
Comment (fgimenez) [inline test/subscriber.ts:55]: Given that the promise is not resolving to any value, why not `Promise<void>` instead?

Comment (ironoa) [reply test/subscriber.ts:55]: it was auto inferred by my IDE ( visual studio code ). Let me quickly check

**Pull request merged**